datalad/__main__.py

```diff
@@ -73,6 +72,21 @@ def main(argv=None):
             sys.stdout.write("datalad %s\n" % __version__)
             sys.exit(0)
 
+    try:
+        from datalad_deprecated.auto import AutomagicIO
+    except Exception as e:
+        # we could just test for ModuleNotFoundError (which should be
+        # all that would happen with PY3.6+, but be a little more robust
+        # and use the pattern from duecredit
+        if type(e).__name__ not in ('ImportError', 'ModuleNotFoundError'):
+            lgr.error("Failed to import datalad_deprecated.auto "
+                      "due to %s", str(e))
+        sys.stderr.write("{}\n".format(
+            "The DataLad AutomagicIO functionality has been moved to an "
+            "extension package. Please install the Python package "
```

Comment:
```suggestion
            "extension package. Please install or update the Python package "
```

Reply:
I think this is fine without the change. There is only a single release of -deprecated, and it has all that is needed. If there is a problem, the message issued by `create-sibling` also needs to be adjusted.

Reply:
Yeah, it's okay. Just thought might be slightly better to make that a "habit" to not require adjustment.